vscode-udfs/README.md

```diff
@@ -0,0 +1,93 @@
+<!-- title: How to create UDFs with Confluent for VS Code -->
+<!-- description: In this tutorial, learn how to develop, upload, register and execute Flink UDFs with Confluent for VS Code, with step-by-step instructions. -->
+
+# VSCode Flink UDFs
```

Comment:
change this title to match what you go with above. `How to use Confluent for VS Code to create a sample UDF in Confluent Cloud for Apache Flink` is long but we're putting together 2 product names so I think it's ok. we want to avoid people thinking that the tutorial is for OSS AF so it should call these things out